product_docs/docs/migration_portal/4.0/04_mp_migrating_database/02_mp_schema_assessment.mdx

```diff
@@ -85,3 +85,41 @@ You can optionally select **Generate PDF** to save the report in PDF format. You
     Migration Portal doesn't assess sensitive PL/SQL source code hidden in Oracle wrapped objects. These wrapped objects aren't included in the assessed objects count, and therefore the true value of compatibility percentage might be different from the value calculated in the assessment report.  
     
 See the note in [Performing a schema extraction](01_mp_schema_extraction#wrapped-objects) for more information about wrapped objects.
+
+## Evaluate an assessment report
+
+The assessment report presents data and information about the compatibility of objects and features in your schemas and the overall project. 
+
+- Object: A CREATE or ALTER TABLE  statement that adds a constraint in the uploaded DDL.
```

Comment:
This wording makes it seem as an object only corresponds to a statement (either CREATE or ALTER TABLE)  that adds a constraint.  All CREATE statements that are parsed by the migration portal results in an object.  In addition, ALTER TABLE statements that add constrains also result in an object.